.circleci/config.yml

```
@@ -30,7 +30,7 @@ jobs:
           command: |
             git remote add upstream https://github.com/pingcap/docs-cn.git
             git fetch upstream
-            markdownlint $(git diff-tree --name-only --no-commit-id -r upstream/master..HEAD -- '*.md' ':(exclude).github/*' ':(exclude)v1.0/*' ':(exclude)v2.0/*' ':(exclude)v2.1-legacy/*')
+            markdownlint $(git diff-tree --name-only --no-commit-id -r upstream/release-4.0..HEAD -- '*.md' ':(exclude).github/*' ':(exclude)v1.0/*' ':(exclude)v2.0/*' ':(exclude)v2.1-legacy/*')
```

```suggestion
            markdownlint $(git diff-tree --name-only --no-commit-id -r upstream/release-4.0..HEAD -- '*.md' ':(exclude).github/*')
```
 @aylei ':(exclude)v1.0/*' ':(exclude)v2.0/*' ':(exclude)v2.1-legacy/*') 这仨文件夹在本仓库已经不存在了，可以删去了不？另外由于这是 release-4.0 分支，所以我把 upstream/master..HEAD 改成了 upstream/release-4.0..HEAD。

---

嗯，LGTM

---

Thanks!